test/CMakeLists.txt

```diff
@@ -46,12 +46,14 @@ add_executable(
     gtest/aca_test_ovs_l2.cpp
     gtest/aca_test_ovs_l3.cpp
     gtest/aca_test_ovs_util.cpp
+    gtest/aca_test_oam.cpp
+    gtest/aca_test_zeta_programming.cpp
 )
 
 # Link test executable against gtest & gtest_main
 target_link_libraries(aca_tests gtest gtest_main)
 target_link_libraries(aca_tests pulsar)
-#target_link_libraries(aca_tests openvswitch)
+target_link_libraries(aca_tests openvswitch)
```

Comment:
not sure if we really need the below, I have comment it out:
# target_link_libraries(aca_tests openvswitch)